src/server/debug/server/debugstar/debugstar.go

```diff
@@ -65,8 +90,13 @@ func (r *reader) Unpack(v starlark.Value) error {
 		r.Reader = strings.NewReader(string(x))
 	case io.Reader:
 		r.Reader = x
+	case starlark.NoneType:
+		r.skip = true
+	case error:
+		r.Reader = strings.NewReader(x.Error())
+		r.err = x
```

Comment:
A bit surprising that in the case of an error, the reader carries both the text of the error and the error. It looks like this is just tested for presence; should this be `isError bool`?